pydantic_extra_types/internal/representation.py

```diff
@@ -0,0 +1,116 @@
+"""
```

Comment:
humm, I'd rather not duplicate this. I think it's okay to use `._internal` from pydantic, we can run this projects unit tests as part of pydantic's tests and hence avoid breaking anything.

Reply:
Yes, cause i can't inherit the `._internal` directory from pydantic to re-use it

Reply:
Within this project, I think it's fine to use 

```py
from pydantic._internal import ...
```

Reply:
either that, or we decide which bits of `_internal` need to be public and move them to a new module.

Reply:
It's can be awesome if we move them to be public and use them as module, that's can help to not break several things, or we will be depending on the repository directly cause I can't find them when I install pydantic

Reply:
The reason you can't find them in pydantic is because you're depending on v1.10, you need to install pydantic from git@main, then `from pydantic._internal import ...` should work.

Reply:
Yes, so i can move with this way to install it

Reply:
Done 🚀